vmm_core/virt_hvf/src/lib.rs

```diff
@@ -1,11 +1,12 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+#![cfg(all(target_os = "macos", target_arch = "aarch64"))] // xtask-fmt allow-target-arch sys-crate
```

Comment:
any reason this line got hoisted up, but the `expect` didn't?

Reply:
working around rustfmt weirdness where it kept trying to put the xtask-fmt allow comment on the next line. And putting #[rustfmt::skip] on things didn't help.

Reply:
that's... funky.

is this the only place this crops up? still reviewing the PR...

if it is - I'd leave a quick comment pointing out the intent behind the inconsistency

Reply:
Yeah it was the only one. I don't know that it needs a comment, the attributes still work fine, just the positioning is a little odd.

Reply:
ok. might want to see if there's a min repro you can send upstream to rustfmt